catalog/CSIC/CSIC_SPEI_2_8.jsonnet

```diff
@@ -0,0 +1,330 @@
+local id = 'CSIC/SPEI/2_8';
+local subdir = 'CSIC';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+local units = import 'units.libsonnet';
+
+local license = spdx.proprietary;
+
+local basename = std.strReplace(id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+{
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+    ee_const.ext_ver,
+  ],
+  id: id,
+  title: 'SPEIbase: Standardised Precipitation-Evapotranspiration Index database, Version 2.8',
+  version: '2.8',
+  'gee:type': ee_const.gee_type.image_collection,
+  description: |||
+    The Global SPEI database, SPEIbase, offers long-time, robust information about drought 
+    conditions at the global scale, with a 0.5 degree pixel size and monthly cadence.
+    It provides SPEI time scales from 1 to 48 months.
+    
+    The Standardized Precipitatin-Evapotranspiration Index (SPEI) expresses, as a standardized variate
+    (mean zero and unit variance), the deviations of the current climatic balance (precipitation minus evapotranspiration potential)
+    with respect to the long-term balance. The reference period for the calculation, in the SPEIbase,
+    corresponds to the whole study period. Being a standardized variate means that the SPEI condition can be compared across space and time.
+    
+    The SPEIbase is based on the FAO-56 Penman-Monteith estimation of potential evapotranspiration.
+    This is a major difference with respect to the SPEI Global Drought Monitor, that uses the Thornthwaite PET estimation.
+    The Penman-Montheith method is considered a superior method, so the SPEIbase is recommended for most uses including long-term climatological analysis.
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id) + [
+    {
+      rel: ee_const.rel.cite_as,
+      href: 'https://digital.csic.es/handle/10261/288226',
+    },
+  ],
+  keywords: [
+    'climate',
+    'climate_change',
+    'drought',
+    'evapotranspiration',
+    'geophysical',
+    'global',
+    'monthly',
+    'palmer',
+    'precipitation',
+    'temperature',
+  ],
+  providers: [
+    ee.producer_provider('Spanish National Research Council (CSIC)','https://spei.csic.es/'),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  extent: ee.extent_global('1901-01-01T00:00:00Z', '2021-01-01T00:00:00Z'),
+  summaries: {
+    gsd: [
+      55660.0,
+    ],
+    'eo:bands': [
+      {
+        name: 'SPEI_01_month',
+        description: 'Standardized Precipitation-Evapotranspiration Index (SPEI) where precipitation and evapotranspiration data was accumulated over the previous month',
```

Comment:
Fit to 80 columns. For short entries like this, use `'string ' +`.  For longer use `|||`. Here that would be

```
       description:
	  'Standardized Precipitation-Evapotranspiration Index (SPEI) where ' +
	  'precipitation and evapotranspiration data was accumulated over the ' +
	  'previous month',
```